src/main/java/com/cloudbees/jenkins/support/impl/LoadStats.java

```diff
@@ -163,9 +168,31 @@ public void writeTo(OutputStream os) throws IOException {
         private final long time;
         private final long clock;
 
+        public CsvContent(String name, String[] tokens, LoadStatistics stats, MultiStageTimeSeries.TimeScale scale) {
+            super(name, tokens);
+
+            time = System.currentTimeMillis();
+            clock = scale.tick;
+            data = new TreeMap<String, float[]>();
+            for (Field f : FIELDS) {
+                try {
+                    MultiStageTimeSeries ts = (MultiStageTimeSeries) f.get(stats);
+                    if (ts != null) {
+                        TimeSeries series = ts.pick(scale);
+                        if (series != null) {
+                            data.put(camelCaseToSentenceCase(f.getName()), series.getHistory());
+                        }
+                    }
+                } catch (IllegalAccessException e) {
```

Comment:
It's a copy/paste from the other constructor, so the lack of documentation was already there. I did the copy/paste because the fields are `final` and I can't use an auxiliary method.